src/widgets/Dialogs.js

```diff
@@ -325,6 +328,10 @@ define(function (require, exports, module) {
             // Remove our global keydown handler.
             KeyBindingManager.removeGlobalKeydownHook(keydownHook);
 
+
+            // Restore previous focus
+            lastFocus.focus();
```

Comment:
`document.activeElement` could be null instead of something else some rare cases, so it might be good to wrap this inside a `null` check

``` js
if (lastFocus) {
  lastFocus.focus();
}
```